spec/controllers/state_file/questions/email_address_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe StateFile::Questions::EmailAddressController do
+  describe ".show?" do
+    context "when contact preference is email" do
+      it "returns true" do
+        intake = create(:state_file_az_intake, contact_preference: "email")
+        expect(described_class.show?(intake)).to eq true
+      end
+    end
+
+    context "when contact preference is not email" do
+      it "returns false" do
+        intake = create(:state_file_az_intake)
+        expect(described_class.show?(intake)).to eq false
+      end
+    end
+  end
+
+  describe "#update" do
+    xit "enqueues job to send the verification code" do
+    end
+  end
```

Comment:
I left some skipped tests in place to signal the future functionality I'll build in a different commit